### PR TITLE
Anisotropic kernel for the multi-fidelity co-Kriging model (MFCK)

### PR DESCRIPTION
**Pull request by mcastanoUQ**
Adding anisotropic kernel computation for the multi-fidelity co-Kriging approach proposed (MFCK) to deal with non-nested DoEs—additional modification adding a regularization parameter in the Negative Marginal Likelihood.

### CODE REVIEW

**Comment by relf** [comment]
Please update title and description to reflect the PR content.

**Comment by mcastanoUQ** [comment]
It's not necessary to have those values. I'll update the PR with the default values of Cobyla

**Review by relf — CHANGES_REQUESTED** [review]
After the commented code removal, I will merge if you have nothing to add.

**Pull request merged**
